server/user-groups.ts

```diff
@@ -56,7 +56,9 @@ export abstract class Auth extends Map<ID, GroupSymbol | ''> {
 		return super.get(user) || Auth.defaultSymbol();
 	}
 	isStaff(userid: ID) {
-		return this.has(userid) && this.get(userid) !== '+';
+		if (!this.has(userid)) return false;
+		const auth = this.get(userid);
+		return auth !== '+' && auth !== Auth.defaultSymbol();
```

Comment:
Oh, we need to hardcode Prize Winner now.

```suggestion
		return auth !== '+' && auth !== '^' && auth !== Auth.defaultSymbol();
```

Maybe the individual config entries should mark `isStaff`?